moderation/preset-reasons.md

```diff
@@ -7,13 +7,13 @@ nav_order: 3
 
 # Preset Reasons
 
-Save your moderators' time by storing preset reasons for them to use when making redundant moderation actions. 
+Save your moderators' time by storing preset reasons to use when making redundant moderation actions. 
 {: .fs-6 .fw-300 }
 
 ---
 
 ## About
-Preset reasons allow for moderators to save time when issuing punishments in a server by using preset flags for rules that are repeatedly violated. 
+Preset reasons allow for moderators to save time when issuing punishments by using preset flags for rules that are repeatedly violated. Like canned responses for moderators
```

Comment:
```suggestion
Preset reasons allow for moderators to save time when issuing punishments by using preset flags for reasons that are repeatedly used (Like canned responses for moderators!)
```

Reply:
Wanted to find a way to incorporate it better, and you've done just that!